### ISSUE TITLE
`hyprctl -j activewindow` outputs invalid json (trailing comma)

### ISSUE THREAD

**Issue by herlev** [hyprwm/Hyprland#1008]
```
$ hyprctl version
Hyprland, built from branch main at commit 78a545112aa564f62e945fdd91ce3f33698bf9b1dirty (?).
flags: (if any)
```

Running `hyprctl -j activewindow` outputs the following:
```
{
    "address": "0x60291c40",
    "at": [1920, 30],
    "size": [2560, 1410],
    "workspace": {
        "id": 4,
        "name": "4"
    },
    "floating": false,
    "monitor": 1,
    "class": "kitty",
    "title": "~",
    "pid": 22758,
    "xwayland": false,
    "pinned": false,
    "fullscreen": false,
    "fullscreenMode": 0
},
```

### Expected outcome
`hyprclt -j activewindow` should output valid json (no trailing comma).

I believe this has been introduced in a recent commit, but don't know exactly which one.

**Comment by vaxerski** [comment]
blame @ElJeffe

**Comment by vaxerski** [comment]
fixed in 5a00f0c657fbfca0ab4fbd5f388042d6084a5997

**Comment by ElJeffe** [comment]
sorry :(
